blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -176,3 +179,8 @@ def test_distinct_on(sql):
     FROM {tbl}) AS anon_1 ORDER BY anon_1."A" ASC
     """.format(tbl=sql.name))
     assert odo(computation, tuple) == (('a', 1), ('b', 2))
+
+
+def test_function_in_expr():
+    s = symbol('s', 'int32')
+    assert odo(compute(s + 1, sa.func.max(1, bind=resource(db_url))), int) == 2
```

Comment:
I'm hesitant to merge this because it seems like we're working around an odo issue in blaze. For example, the following should be okay for blaze:

```python
s = symbol('s', 'int32')
sqla = compute(s + 1, sa.func.max(1))
odo(sqla, int)  # this fails, but that's because there's no connection
```

Should we instead do something like `odo(sqla, int, bind=engine)`?